src/assets/styles/global.css

```diff
@@ -0,0 +1,147 @@
+:root {
+  --white-bg-color: #ffffff;
+  --main-blue-color: #343951;
+  --secondary-bg-color: #f5f6fc;
+  --sidebar-tooltip-bg-color: #464c61;
+  --main-yellow-color: #ff9f00;
+  --secondary-yellow-color: #ffb233;
+  --main-green-color: #4ac29d;
+  --blue-color: #689ff8;
+  --main-red-color: #ff6860;
+  --font-color: #858a9d;
+  --icon-gray-color: #c4c4c4;
+  --circle-outline-color: #bec0c8;
+  --grey-fill-color: #bcbeca;
+  --linear-gradient: linear-gradient(
+    90deg,
+    rgba(255, 159, 0, 1) 0%,
+    rgba(255, 104, 96, 1) 50%,
+    rgba(104, 158, 248, 1) 100%
+  );
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  height: 100%;
+  min-height: 100vh;
+  margin: 0;
+  display: grid;
+  grid-template-columns: 1fr 3.27fr 1.47fr; /*172px 562px 252px*/
+  gap: 16px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+main {
+  background-color: var(--secondary-bg-color);
+}
+
+nav {
+  gap: 16px;
+}
+
+#root {
+  display: contents;
+}
+
+.yellow-box,
+.blue-box,
+.green-box,
+.grey-box,
+.red-box {
+  display: block;
+  width: 28px;
+  height: 28px;
+  border-radius: 8px;
+}
+
+.yellow-box {
+  background-color: var(--main-yellow-color);
```

Comment:
in order to avoid this, should be better just pass the backgroud-color in an inline-style while rendering the box?

Reply:
That's definitely a better approach, will keep it in mind for my next projects.